common/LongPathHandler_test.go

```diff
@@ -0,0 +1,31 @@
+package common
+
+import (
+	chk "gopkg.in/check.v1"
+)
+
+type pathHandlerSuite struct{}
+
+var _ = chk.Suite(&pathHandlerSuite{})
+
+func (p *pathHandlerSuite) TestShortToLong(c *chk.C) {
+	if OS_PATH_SEPARATOR == `\` {
+		c.Assert(ToLongPath(`C:\myPath`), chk.Equals, `\\?\C:\myPath`)
+		c.Assert(ToLongPath(`\\myHost\myPath`), chk.Equals, `\\?\UNC\myHost\myPath`)
+		c.Assert(ToLongPath(`\\?\C:\myPath`), chk.Equals, `\\?\C:\myPath`)
+		c.Assert(ToLongPath(`\\?\UNC\myHost\myPath`), chk.Equals, `\\?\UNC\myHost\myPath`)
+	} else {
+		c.Skip("Test only pertains to Windows.")
+	}
```

Comment:
If LongPathHandler continues to also do replacements between / and \, then these tests should cover that.